topostats/tracing/dnatracing.py

```diff
@@ -401,11 +402,7 @@ def get_splined_traces(self):
         This function actually calculates the average of several splines which is important for getting a good fit on
         the lower res data"""
 
-        step_size = int(7e-9 / (self.pixel_size))  # 3 nm step size
-        interp_step = int(1e-10 / self.pixel_size)
-        # Lets see if we just got with the pixel_to_nm_scaling
-        # step_size = self.pixel_size
-        # interp_step = self.pixel_size
+        step_size_px = int(self.step_size_m / (self.pixel_size))
 
         # FIXME : Iterate over self.fitted_traces directly use either self.fitted_traces.values() or self.fitted_trace.items()
         for dna_num in sorted(self.fitted_traces.keys()):
```

Comment:
I would recommend iterating over `self.fitted_traces.items()` using as otherwise I think `pylint` might complain about this, but realise this is outside the scope of your additions (but I started writing this before clocking which method this was under).

```suggestion
        for dna_num, _ in sorted(self.fitted_traces.items()):
```